tracer/src/Datadog.Trace.Bundle/Datadog.Trace.Bundle.targets

```diff
@@ -0,0 +1,77 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="RemoveDatadogNuGetFiles" AfterTargets="Publish" Condition="'$(RemoveUnusedDatadogFiles)' == 'true'">
+    <PropertyGroup>
+      <!-- Root where Datadog bundle files are published -->
+      <BundleRoot>$(PublishDir)datadog\</BundleRoot>
+
+      <!-- Helper properties -->
+      <IsNetFramework>$(TargetFramework.StartsWith('net4'))</IsNetFramework>
+
+      <!-- Map TargetFramework to required folder -->
+      <EffectiveTargetFramework Condition="$(IsNetFramework)">net461</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'netstandard2.0'">netstandard2.0</EffectiveTargetFramework>
```

Comment:
Can we add `netstandard2.1` as well?

Reply:
Done, thanks!

Reply:
I targeted `netstandard2.0` for now since we don't know which runtime the app will run in. It could be anything .NET Core 3.0 or above. If it was .NET Core 3.1 or above, we could use `netcoreapp3.1` instead.

Alternatively, we can remove `net461` only and keep `netstandard2.0, `netcoreapp3.1`, and `net6.0`?

Reply:
> Alternatively, we can remove `net461` only in this case and keep `netstandard2.0`, `netcoreapp3.1`, and `net6.0` to keep the performance optimizations for each platform?

Same goes for `netstandard2.0`, now that I think about it.

Reply:
> But maybe what we should do is remove net461 only in this case and keep netstandard2.0, netcoreapp3.1, and net6.0 to keep the performance optimizations for each platform. Same goes for netstandard2.0, now that I think about it (but keeping net461 as well in that case).

Potentially, but really in what situation are you going to be _publishing_ to .NET Standard 🤔 i.e. in what situation would we need those TFMs but _not_ know it "now"?

I guess the main answer is if the target platform has roll forward configured 🤔 In fact, that could be a genuine issue anyway - where the target platform is _not_ what is used to actually run the application... That's not an issue if they're doing a self-contained deployment, but it _could_ be for a framework-dependent deployment 🤔

Reply:
> really in what situation are you going to be publishing to .NET Standard

You can publish a class library project that targets `netstandard2.0`, but yeah, nobody should rely on that for deploy the `dd-trace-dotnet` files. Should we consider `netstandard*` as "not supported" and error out? Or just not copy the `dd-trace-dotnet` files at all?

>  if the target platform has roll forward configured... where the target platform is not what is used to actually run the application

You mean something like this? Your app targets `net5.0`, so they get the `netcoreapp3.1` files when you publish. Then you run your app in `net8.0`, and the tracer looks for the `net6.0` files, which are not there.

Reply:
Have we considered adding the various `Datadog.Trace.dll` files to the nuget as "normal" managed library references instead of content in the `datadog/net*` folder?